source-gallery/src/main/java/com/google/android/apps/muzei/gallery/GalleryImportPhotosDialogFragment.java

```diff
@@ -0,0 +1,110 @@
+package com.google.android.apps.muzei.gallery;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.pm.ActivityInfo;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AlertDialog;
+import android.widget.ArrayAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GalleryImportPhotosDialogFragment extends DialogFragment {
+    public static final String TAG = GalleryImportPhotosDialogFragment.class.getSimpleName();
+
+    private List<ActivityInfo> mGetContentActivities = new ArrayList<>();
+    private OnRequestContentListener mListener;
+    private ArrayAdapter<CharSequence> mAdapter;
+
+    public static GalleryImportPhotosDialogFragment createInstance() {
+        return new GalleryImportPhotosDialogFragment();
+    }
+
+    public GalleryImportPhotosDialogFragment () {
+    }
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        mAdapter = new ArrayAdapter<>(getSafeContext(), android.R.layout.select_dialog_item);
```

Comment:
Please pull it from the theme:

    final TypedArray a = getContext().obtainStyledAttributes(null, R.styleable.AlertDialog,
                R.attr.alertDialogStyle, 0);
    @LayoutRes int listItemLayout = a.getResourceId(R.styleable.AlertDialog_listItemLayout, 0);
    a.recycle();